packages/top-nav/src/TopNavItem.ts

```diff
@@ -73,10 +85,22 @@ export class TopNavItem extends LikeAnchor(Focusable) {
 
     protected override firstUpdated(changes: PropertyValues): void {
         super.firstUpdated(changes);
+        // @TODO - refactor this as a ResizeObserver up to `sp-tabs` so that it can be more
```

Comment:
This TODO, copied from the same workflow in the Tab element, depends on feedback to https://github.com/lit/lit/issues/3237. Depending on which way that convo goes, we'll either move forward with the Resize Controller or need to do something custom with our own Resize Observer.